src/test/kotlin/LottoMachineTest.kt

```diff
@@ -0,0 +1,35 @@
+package lotto
+
+import org.junit.jupiter.api.Test
+import kotlin.test.DefaultAsserter.assertEquals
+
+class LottoMachineTest {
+    @Test
+    fun `should return right amount of tickets`() {
+        val lottoMachine = LottoMachine()
+        assertEquals("Amount isn't a multiple of 1000", lottoMachine.createTickets(5000).count(), 5)
+    }
+
+    @Test
+    fun `ticket should contain 6 numbers`() {
+        val lottoMachine = LottoMachine()
+        val tickets = lottoMachine.createTickets(1000)
+        val numbersInTicket = tickets[0].getNumbers()
+        assertEquals("Ticket numbers aren't 6", numbersInTicket.count(), 6)
+    }
+
+    @Test
+    fun `amount of tickets were generated`() {
+        val lottoMachine = LottoMachine()
+        assertEquals("Amount of tickets is different then expected", lottoMachine.createTickets(4000).count(), 4)
+    }
+
+    @Test
+    fun `correct amount of rate is calculated`() {
+        val lottoMachine = LottoMachine()
+        val results = mutableMapOf<Rank, Int>()
+        results[Rank.FIFTH] = 1
+        val amount = 3000
+        assertEquals("Rate result is different then expected", lottoMachine.calculateReturnRate(results, amount), 1.6666666666666667)
```

Comment:
```suggestion
        assertEquals(
            actual = lottoMachine.calculateReturnRate(results, amount),
            expected = 1.66666666666667,
            message = "Rate result is different than expected"
        )
```

Although it might look like they bring the same results,
putting values into their respected place is important for logic to work as expected.

One way to achieve this is by using [named parameters](https://kotlinlang.org/docs/functions.html#named-arguments) just like the example above.
Before it, `1.6666666666666667` was in the `actual` spot and `lottoMachine.calculate...` was in the `actual`. Also, notice that I put the message at the very bottom since it's the least important part of this assertion.

Let's try refactoring the test logic :)